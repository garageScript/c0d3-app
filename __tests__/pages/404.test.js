import React from 'react'
import { render, waitFor } from '@testing-library/react'
import { MockedProvider } from '@apollo/client/testing'
import NotFound from '../../pages/404'

describe('404 Error Page', () => {
  it('should render', async () => {
    const { container } = render(
      <MockedProvider>
        <NotFound />
      </MockedProvider>
    )
    await waitFor(() => expect(container).toMatchSnapshot())
  })
})
