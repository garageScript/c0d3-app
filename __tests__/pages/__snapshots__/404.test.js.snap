// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 Error Page should render 1`] = `
<div>
  <nav
    class="navbar navbar-expand-lg navbar-light justify-content-between bg-white"
  >
    <div
      class="container"
    >
      <a
        class="navbar-brand text-primary font-weight-bold"
        href="/"
      >
        C0D3
      </a>
      <div
        id="navbarNav"
      >
        <div
          class="navbar-nav collapse navbar-collapse"
        >
          <div
            class="navbar-nav collapse navbar-collapse"
          >
            <a
              class="nav-item nav-link"
              href="/curriculum"
            >
              Curriculum
            </a>
            <a
              class="nav-item nav-link"
              href="https://github.com/garageScript/c0d3-app"
              rel="noopener noreferrer"
              target="_blank"
            >
              Repo
            </a>
            <a
              class="nav-item nav-link"
              href="https://www.notion.so/Table-of-Contents-a83980f81560429faca3821a9af8a5e2"
              rel="noopener noreferrer"
              target="_blank"
            >
              Journey
            </a>
            <a
              class="nav-item nav-link"
              href="https://chat.c0d3.com"
              rel="noopener noreferrer"
              target="_blank"
            >
              Help
            </a>
            <a
              class="nav-item nav-link"
              href="/contributors"
            >
              Contributors
            </a>
          </div>
        </div>
      </div>
      <div>
        <a
          class="btn btn-secondary border mr-3"
          href="/login"
        >
          Login
        </a>
        <a
          class="btn btn-secondary border mr-3"
          href="/signup"
        >
          Signup
        </a>
      </div>
    </div>
  </nav>
  <div
    class="container"
  >
    <div
      class="container"
    >
      <div
        class="row"
      >
        <div
          class="d-flex col-sm-3 align-items-center justify-content-center"
        >
          <div
            class="text-center mt-3"
          >
            <div
              style="color: rgb(41, 41, 41); font-size: 1.5rem;"
            >
              <strong>
                Page not found
              </strong>
            </div>
            <div
              style="color: rgb(41, 41, 41); font-size: 1rem;"
            >
              Page not found
            </div>
            <a
              class="btn btn-primary py-3 px-5 mt-3"
              href="/"
            >
              <h3
                class="font-weight-bold"
              >
                Back
              </h3>
            </a>
          </div>
        </div>
        <div
          class="mb-3 col-sm-9"
        >
          <div
            style="display: block; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
          >
            <div
              style="display: block; box-sizing: border-box; padding-top: 100%;"
            />
            <img
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="visibility: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: contain;"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
  <footer
    class="mt-5 font-weight-light text-center pb-5 text-muted"
  >
    Â© Copyright 2020 GarageScript
    <br />
    All Rights Reserved
  </footer>
</div>
`;
